src/App.jsx

```diff
@@ -1,14 +1,82 @@
 import './App.css';
+import messagesDataDefault from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
+import ColorComponents from './components/ColorComponents';
 
 const App = () => {
+  const [messagesData, setMessageData] = useState(messagesDataDefault);
+  const [likeCount, setLikeCount] = useState(0);
```

Comment:
We can calculate the number of likes without introducing a piece of state. We should prefer to calculate a value instead of using state where possible to reduce an app's complexity.